### DISCUSSION TITLE
How to express uncertainty in year data

### DISCUSSION THREAD

**Discussion by jocelynpender** [atlanticcanadacdc/template#40] [Q&A]
Received raw data has `2021?` as date.
Does this get normalized into `2021`? Do we express our uncertainty about the data?

**Comment by jlchurchill** [comment]
has this been resolved? (based on text that now is in the DD). When the ? is after the full year value it is difficult to know at what digits the uncertainty in the date lies. My guess is that it is a relatively recent record so the uncertainty might only be in one of the last two digits but difficult to know without checking with the observer. A safe bet might be to use 20** and put 2021? in obDateVerbatim.
